doc/API.md

```diff
@@ -908,9 +908,9 @@ funds // hastings
 hosts
 period      // block height
 renewwindow // block height
-maxdownloadspeed  // bytes per second, not persisted and will be reset by a shutdown
-maxuploadspeed  // bytes per second, not persisted and will be reset by a shutdown
-streamcachesize // number of data chunks cached when streaming, not persisted and will be reset by a shutdown
+maxdownloadspeed  // bytes per second
+maxuploadspeed  // bytes per second
+streamcachesize // number of data chunks cached when streaming
```

Comment:
non-blocking, but we like to align the comments if they are all in a row, example:

```
abc     // comment 1
defghij // comment 2
lmop    // comment 3
```